src/lib/crypto/snarky_tests/snarky_tests.ml

```diff
@@ -666,16 +664,25 @@ let api_tests =
   ]
 
 let () =
+  Async.Thread_safe.block_on_async_exn
+  @@ fun () ->
   let range_checks =
     List.map ~f:QCheck_alcotest.to_alcotest [ RangeCircuits.test_range_gates ]
   in
+  let logger = Logger.create () in
+  let%map.Async.Deferred constraint_constants =
+    let%map.Async.Deferred config =
+      Runtime_config.Constants.load_constants ~logger []
```

Comment:
We were previously using the compiled constraints, we can change them but then the test is different. For the record, this is what happened when we broke the snarky profile tests.

As for the `[]`, the deferred is needed because we are still trying to load files -- this is because we are always reading for files that are found (1) via debian install (2) in the `mina_conf` dir, etc.